### PR TITLE
fix: temporarily disable telemetry on windows

### PR DESCRIPTION
**Pull request by jtoar**
"Closes" https://github.com/redwoodjs/redwood/issues/5757, https://github.com/redwoodjs/redwood/issues/5728. This isn't a fix, but the behavior is annoying enough that we should've disabled it a long time ago. (So we're disabling it now!)

At first I thought about making these changes in `@redwoodjs/cli`, but doing it in `@redwoodjs/telemetry`ended up being simpler.

### CODE REVIEW

**Review by dac09 — APPROVED** [review]

**Comment by dac09** [comment]
Created this issue so we don't lose track. https://github.com/redwoodjs/redwood/issues/5901

File: packages/telemetry/src/telemetry.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { spawn } from 'child_process'
+import os from 'node:os'
```
Comment (dac09) [inline packages/telemetry/src/telemetry.ts:2]: I'd be careful these - I'm not sure why... but I've found there are some cases where it throws errors (on Windows... if I remember correctly) - and other places the same imports didn't.

**Comment by dac09** [comment]
@jtoar - I can't the CI tests to pass ... I wonder if something is actually breaking after this change?

**Comment by jtoar** [comment]
@dac09 I think CI is just a little flaky sometimes. I just gave it a kick and it's all green now. Here's the latest run: https://github.com/redwoodjs/redwood/actions/runs/2655619534.

**Comment by dac09** [comment]
@jtoar as I mentioned in this PR - looks like the fancy node import causes issues. 

```
PS C:\Users\dac09\dev\telemetry-test> yarn rw info
yarn run v1.22.11
$ C:\Users\dac09\dev\telemetry-test\node_modules\.bin\rw info
internal/modules/cjs/loader.js:892
  throw err;
  ^

Error: Cannot find module 'node:os'
Require stack:
- C:\Users\dac09\dev\telemetry-test\node_modules\@redwoodjs\telemetry\dist\telemetry.js
- C:\Users\dac09\dev\telemetry-test\node_modules\@redwoodjs\telemetry\dist\index.js
- C:\Users\dac09\dev\telemetry-test\node_modules\@redwoodjs\cli\dist\index.js
- C:\Users\dac09\dev\telemetry-test\node_modules\@redwoodjs\cli\package.json
-  ```

**Pull request merged**